tests/providers/amazon/aws/operators/test_redshift_pause_cluster.py

```diff
@@ -52,10 +52,8 @@ def test_init(self):
         redshift_operator = RedshiftPauseClusterOperator(
             task_id="task_test",
             cluster_identifier="test_cluster",
-            aws_conn_id="aws_conn_test",
-            check_interval=3,
+            aws_conn_id="aws_conn_test"
```

Comment:
You have a default value in the Operator for this, am I missing a reason to have it defined here?  Looks like an opportunity to just use the default.